gcip/addons/container/config.py

```diff
@@ -131,6 +132,6 @@ def get_shell_command(self) -> List[str]:
         """
         script = [
             f"mkdir -p {self._config_file_path}",
-            "echo " + '"' + json.dumps(self.config).replace('"', '\\"') + '"' + f" > {self._config_file_path}{self._config_file_name}",
+            "echo " + '"' + json.dumps(self.config).replace('"', '\\"') + '"' + f" > {os.path.join(self._config_file_path, self._config_file_name)}",
```

Comment:
After using `os.path.join` we don't need that bulky f-String anymore.

```suggestion
            'echo "' + json.dumps(self.config).replace('"', '\\"') + '" > ' + os.path.join(self._config_file_path, self._config_file_name),
```

Pls test the suggestion after applying it.

Reply:
Thank you for the hint.
I did ```'echo "' + json.dumps(self.config).replace('"', '\\"') + '" > ' + os.path.join(self._config_file_path, self._config_file_name)```

May you approve?